src/com/andersen/lesson2/TicketService.java

```diff
@@ -26,4 +52,12 @@ public static void save_price(int price, @NotNull Ticket t){
         System.out.println("Price is saved.");
     }
 
+    private static Ticket getTicketByStadiumSector(Collection<Ticket> tickets, char stadiumSector) {
+        for (final var ticket : tickets) {
+            if (ticket.getStadiumSector() == stadiumSector) {
+                return ticket;
+            }
+        }
+        return null;
```

Comment:
It is not a best practice to return null here, better just throw exception if ticker is null either use Optional as return type for this method